# This expresses velocity in free space broken into it's linear and angular parts. Also, the distance left to leave the sheet
geometry_msgs/Vector3  linear
geometry_msgs/Vector3  angular
float32  distance_left
